site/docs/quarks-getting-started.md

```diff
@@ -25,9 +25,14 @@ Quarks' primary API is functional where streams are sourced, transformed, analyz
 
 ### Downloading Apache Quarks
 
-To use Quarks, access the source code and build it. You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+To use Quarks, you can either download the source code and build it yourself, or you can download and install a pre-built release. 
 
-After you build the Quarks package, you can set up your environment.
+You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+
+You can find pre-built releases [here](https://github.com/quarks-edge/quarks/releases).
+Please note:  While Apache Quarks is undergoing incubation at The Apache Software Foundation (ASF), these pre-release builds will be hosted directly on gitub, not at Apache.
```

Comment:
I think it needs to be noted that these are not Apache releases, where they are hosted is not really relevant.